server/sonar-server/src/test/java/org/sonar/server/measure/index/ProjectMeasuresIndexTest.java

```diff
@@ -1183,6 +1185,7 @@ public void facet_languages_is_sticky() {
       entry("500000.0-*", 0L));
   }
 
+  @Ignore
   @Test
   public void facet_languages_returns_more_than_10_languages_when_languages_filter_contains_value_not_in_top_10() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)